yarn.lock

```diff
@@ -12413,13 +12413,28 @@ supports-color@^6.1.0:
   dependencies:
     has-flag "^3.0.0"
 
+supports-color@^7.0.0:
```

Comment:
Did you forget to run Yarn afterwards? Nothing regarding the sourcemap package changed in the lockfile

Reply:
Yup forgot that, updated it